homeassistant/components/nut/sensor.py

```diff
@@ -67,13 +70,21 @@ def __init__(
         sensor_description: SensorEntityDescription,
         data: PyNUTData,
         unique_id: str,
-        enabled_default: bool,
+        enabled_resources: list,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self.entity_description = sensor_description
 
         device_name = data.name.title()
+        if enabled_resources:
+            enabled_default = sensor_description.key in enabled_resources
+        else:
+            enabled_default = (
+                sensor_description.entity_registry_enabled_default
+                and sensor_description.entity_category is None
```

Comment:
I don't think we should check category too. The description attribute enabled default should be the source of truth.

Reply:
Was a shortcut to not set attribute for most entities, but no problem, I will change it.